ui/app/rif/components/tokenNetworkChannels.js

```diff
@@ -0,0 +1,90 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import { GenericTable } from './index';
+import rifActions from '../actions';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+
+class TokenNetworkChannels extends Component {
+
+  static propTypes = {
+    paginationSize: PropTypes.number,
+    getChannels: PropTypes.func,
+    getChannel: PropTypes.func,
+    classes: PropTypes.any,
+  }
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      channelsData: [],
+    };
+  }
+
+  loadData () {
+    const { classes } = this.props;
+    this.getChannels().then(channels => {
+      channels.map((channelJson) => {
+        const channel = channelJson[Object.keys(channelJson)[0]];
+        const tableRow = {};
+        tableRow.content =
+          <div className={classes.content}>
```

Comment:
Same thing here, parenthesis is a good practice when you define jsx react components